lib/pf/Switch/Meraki/AP_http_CoA.pm

```diff
@@ -0,0 +1,418 @@
+package pf::Switch::Meraki::AP_http_CoA;
+
+=head1 NAME
+
+pf::Switch::Meraki::AP_http_CoA
+
+=head1 SYNOPSIS
+
+The pf::Switch::Meraki::AP_http_CoA module implements an object oriented interface to
+manage the external captive portal on Meraki access points
+
+=head1 STATUS
+
+Developed and tested on a MR12 access point
+
+=head1 BUGS AND LIMITATIONS
+
+=head2 Cannot reevaluate the access
+
+There is currently no way to reevaluate the access of the device.
+There is neither an API access or a RADIUS disconnect that can be sent either to the AP or to the controller.
+
+=head2 client IP cannot be computed from parseUrl
+
+The Meraki sends a NATed IP address in the URL even though the client is bridged.
+The only workaround is to have the DHCP traffic forwarded to PacketFence.
+
+=cut
+
+use strict;
+use warnings;
+
+use base ('pf::Switch');
```

Comment:
This module should use pf::Meraki::AP_http as its base.

You are extending the support, not rewriting it